test/util/test_runners/integrations_tests_runner.py

```diff
@@ -41,11 +42,11 @@ def main():
     args = TestRunnerArgs.create_from_args()
 
     runner = TestRunner(args)
-
-    test_to_run_list=[CliIntegrationTest,
-                      CliIntegrationTestConfigurations,
+    test_to_run_list=[CliIntegrationTestConfigurations,
+                      CliIntegrationTestVariable,
                       CliIntegrationTestCredentials,
-                      CliIntegrationListTest]
+                      CliIntegrationTestList,
+                      CliIntegrationPolicy]
```

Comment:
also not for this PR but the name of this file not so good and we also need to move all the test runner files into different location what do you think?

Reply:
agreed maybe in a separate file that as an ENUM or just in a list in consts?

Reply:
I meant this file "integrations_tests_runner.py". my bad naming not you